src/auth/digest/eDirectory/ldap_backend.cc

```diff
@@ -261,15 +261,16 @@ getpassword(char *login, char *realm)
         if (rc == LDAP_SUCCESS) {
             entry = ldap_first_entry(ld, res);
             if (entry) {
-                debug("ldap dn: %s\n", ldap_get_dn(ld, entry));
+                char *dn = ldap_get_dn(ld, entry);
```

Comment:
```suggestion
                const auto dn = ldap_get_dn(ld, entry);
```
(untested, please double check)

Reply:
using char keeps it consistent with other uses of ldap_get_dn, e.g. https://github.com/squid-cache/squid/blob/8f163804ad40ec3beff11c730a45fce082dd7d57/src/auth/basic/LDAP/basic_ldap_auth.cc#L685 and https://github.com/squid-cache/squid/blob/8f163804ad40ec3beff11c730a45fce082dd7d57/src/acl/external/LDAP_group/ext_ldap_group_acl.cc#L766

Reply:
Rule of thumb: Unless there is a specific compelling reason to do otherwise in a specific context, please ignore general consistency concerns when adding AAA and `const` to new or changed variable declarations and parameter definitions. Otherwise, very little code can be improved -- too much of legacy Squid code is inconsistent with modern practices; we need to make progress while we change code for other compelling reasons, especially where that progress does not cause widespread merge conflicts and other serious problems.